.bingo/Variables.mk

```diff
@@ -29,11 +29,11 @@ $(BINGO): $(BINGO_DIR)/bingo.mod
 	@echo "(re)installing $(GOBIN)/bingo-v0.5.1"
 	@cd $(BINGO_DIR) && $(GO) build -mod=mod -modfile=bingo.mod -o=$(GOBIN)/bingo-v0.5.1 "github.com/bwplotka/bingo"
 
-GOLANGCI_LINT := $(GOBIN)/golangci-lint-v1.56.2
+GOLANGCI_LINT := $(GOBIN)/golangci-lint-v1.61.0
```

Comment:
this is an auto-generated file (see header), it's not how it should be updated.
TBH I'm not a fan of "bingo", the tool used for bin dependencies here (I find it a bit opaque for the simple job we're asking .. and I always forget how it works :-/ ) .. let me see if I can quickly remove that and align this repo we our other ones

Reply:
ops, my bad. Good choice removing bingo.